lib/xwax/timecoder.c

```diff
@@ -603,8 +751,16 @@ void timecoder_submit(struct timecoder *tc, signed short *pcm, size_t npcm)
             secondary = left;
         }
 
-	process_sample(tc, primary, secondary);
-        update_monitor(tc, left, right);
+        if (tc->def->flags & TRAKTOR_MK2) {
+            process_sample(tc, primary, secondary);
+
+            /* Display the derivative in the monitor */
+            update_monitor(tc, tc->primary.mk2.deriv_scaled << 1,
+                    tc->secondary.mk2.deriv_scaled << 1);
```

Comment:
why the left bitshift?

Reply:
That is just a multiplication by two. It makes the signal fit nicely into the scope window.

Reply:
Changed and added an explanation